packages/wellknown-did-verifier/__tests__/localAgent.test.ts

```diff
@@ -0,0 +1,76 @@
+import { getConfig } from '@veramo/cli/build/setup'
+import { createObjects } from '@veramo/cli/build/lib/objectCreator'
+import { ServiceTypesEnum } from '@sphereon/wellknown-dids-client/dist/types'
+import wellKnownDidVerifierAgentLogic from './shared/wellKnownDidVerifierAgentLogic'
+
+jest.setTimeout(30000)
+
+let agent: any
+
+const setup = async (): Promise<boolean> => {
+  const config = getConfig('packages/wellknown-did-verifier/agent.yml')
+  const { localAgent } = createObjects(config, { localAgent: '/agent' })
+
+  await localAgent.registerSignatureVerification({
+    key: 'verified',
+    signatureVerification: () => Promise.resolve({ verified: true })
+  }, null)
+
+  const DID = 'did:key:z6MkoTHsgNNrby8JzCNQ1iRLyW5QQ6R8Xuu6AA8igGrMVPUM';
+  const ORIGIN = 'https://example.com';
+  const DOCUMENT = {
+    '@context': [
+        'https://www.w3.org/ns/did/v1',
```

Comment:
Doesn't look like it is formatted with prettier

Reply:
fixed